src/components/asset-list/RecyclerAssetList2/core/ExternalScrollView.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext, useImperativeHandle } from 'react';
+import { Animated as RNAnimated, ScrollViewProps } from 'react-native';
+import BaseScrollView, {
+  ScrollViewDefaultProps,
+} from 'recyclerlistview/dist/reactnative/core/scrollcomponent/BaseScrollView';
+import { useMemoOne } from 'use-memo-one';
+import { useRecyclerAssetListPosition } from './Contexts';
+import { StickyHeaderContext } from './StickyHeaders';
+
+const extraPadding = { paddingBottom: 144 };
+// @ts-ignore
+const ExternalScrollViewWithRef = React.forwardRef(function ExternalScrollView(
+  props: ScrollViewDefaultProps,
+  ref
+): BaseScrollView {
+  const y = useRecyclerAssetListPosition()!;
+
+  const { onScroll, ...rest } = props;
+  const { scrollViewRef } = useContext(StickyHeaderContext)!;
+
+  const event = useMemoOne(
+    () =>
+      RNAnimated.event(
+        [
+          {
+            nativeEvent: {
+              contentOffset: {
+                y,
+              },
+            },
+          },
+        ],
+        { listener: onScroll, useNativeDriver: true }
+      ),
+    [onScroll, y]
+  );
+
+  useImperativeHandle(ref, () => scrollViewRef.current);
+  // @ts-ignore
```

Comment:
This file has a few `@ts-ignore` statements - why are they needed?

Reply:
I don't know, can you maybe take a look?